test/parallel/test-fs-copyfile-respect-permissions.js

```diff
@@ -5,6 +5,9 @@
 
 const common = require('../common');
 
+if (process.getuid() === 0)
```

Comment:
`process.getuid()` doesn’t exist on Windows.

Reply:
Thanks! I've added an `isWindows` check for this now.